sklearnex/glob/__main__.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+#===============================================================================
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+from sklearnex import patch_sklearn
+from sklearnex import unpatch_sklearn
+
+
+def _main():
+    import argparse
+
+    # Adding custom extend action for support all python versions
+    class ExtendAction(argparse.Action):
+        def __call__(self, parser, namespace, values, option_string=None):
+            items = getattr(namespace, self.dest) or []
+            items.extend(values)
+            setattr(namespace, self.dest, items)
+
+    parser = argparse.ArgumentParser(
+        prog="python -m sklearnex.glob",
+        description="""
+            Patch your all Scikit-learn applications using Intel(R) Extension for
+            scikit-learn, optimizing solvers of
+            scikit-learn with Intel(R) oneAPI Data Analytics Library.
+            """,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.register('action', 'extend', ExtendAction)
+    parser.add_argument('action', choices=["patch_sklearn", "unpatch_sklearn"],
+                        help="Enable or Disable patching")
+    parser.add_argument('--no-verbose', '-nv', action='store_false',
+                        help="Additional info about patching")

```

Comment:
```suggestion
                        help="Additional information about the patching")
```

what is the action here? e.g. in line 43 the action is enable/disable, you need a verb here as well.
plus I'd assume that no-verbose means that you are turning off the verbose mode, meaning you are not providing any additional info